src/components/Modal/DesignedModal/DesignedModal.styles.tsx

```diff
@@ -22,12 +22,18 @@ export const Container = styled(BaseModal)`
 
 export const Modal = styled.div<{
   isDraggable?: boolean;
+  width?: number | string;
+  height?: number | string;
```

Comment:
```suggestion
  $width?: number | string;
  $height?: number | string;
```

transient-props 처리만 해주심 좋을 것 같습니다!!

Reply:
@kimjeongwonn 생각해보니 height는 Modal의 children 부분에서 고정값으로 주면될거 같아서 불필요할거 같네요 이거랑 함께 수정할게요